src/main/java/io/papermc/bibliothek/controller/v2/ProjectsController.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of bibliothek, licensed under the MIT License.
+ *
+ * Copyright (c) 2019-2021 PaperMC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.papermc.bibliothek.controller.v2;
+
+import io.papermc.bibliothek.database.model.Project;
+import io.papermc.bibliothek.database.repository.ProjectCollection;
+import io.papermc.bibliothek.util.HTTP;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.CacheControl;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+@SuppressWarnings("checkstyle:FinalClass")
+public class ProjectsController {
+  private static final CacheControl CACHE = HTTP.sMaxAgePublicCache(Duration.ofDays(7));
+  private final ProjectCollection projects;
+
+  @Autowired
+  private ProjectsController(final ProjectCollection projects) {
+    this.projects = projects;
+  }
+
+  @ApiResponse(
+    content = @Content(
+      schema = @Schema(implementation = ProjectsResponse.class)
+    ),
+    responseCode = "200"
+  )
+  @GetMapping("/v2/projects")
+  @Operation(summary = "Gets a list of all available projects.")
```

Comment:
Super tiny nitpick here, you use `Get ...` everywhere else.
```suggestion
  @Operation(summary = "Get a list of all available projects.")
```